docs/guide-hermes-release.md

```diff
@@ -20,14 +32,82 @@ git checkout -b rn/0.76-stable
 git push origin HEAD
 ```
 
-### For stable patch releases
+### Step 2: Cherry-pick
+
+#### For (Legacy) Hermes
+
+> [!Important]
+> If you cutting a release candidate, skip this step
+
+Pick the relevant commits onto that branch. The pick requests should be from `main` and no other branch on Hermes.
+
+Push the picks to the remote branch.
+
+#### For HermesV1
+
+> [!Important]
+> If you cutting a release candidate, skip this step
+
+1. Checkout the `250829098.0.0-stable` branch.
+2. Pick the relevant commits onto that branch. The pick requests should be from `static_h` and no other branch on Hermes.
+3. Push the picks to the remote branch.
+
+> [!Important]
+> If you cherry picked something, you need to bump the patch version of hermes, as shown in [this commit](https://github.com/facebook/hermes/commit/ddd708a85b164d1841c024973d0f6d3fad60a4c2)
+> 1. Open the [`npm/hermes-compiler/package.json`](https://github.com/facebook/hermes/blob/ddd708a85b164d1841c024973d0f6d3fad60a4c2/npm/hermes-compiler/package.json) file
+> 2. Bump the patch number by 1
+> 3. Commit and push.
+
+### Step 3: Publish Tag
+
+This step should be automatic. When a new commit is pushed, we should have a new tag in the Hermes repo.
+
+If that's not it, navigate to the [Publish Tag workflow](https://github.com/facebook/hermes/actions/workflows/create-tag.yml) and runthe workflow twice:
+
+#### For (Legacy) Hermes
+
+1. Set the branch to the release branch (e.g.: rn/0.83-stable)
+2. Set the release type as `Release`
+3. Set the hermes version to follow this pattern: `hermes-<date as YYYY-mm-DD>-RNv<version-of-react-native>`. For example: `hermes-2025-11-03-RNv0.83.0`. Do not add he prerelease tag
+
+#### For Hermes V1
+
+1. Set the branch to the Hermes V1 release branch: `250829098.0.0-stable`
+2. Set the release type as `Release`
+3. Set the hermes version to follow this pattern: `hermesV1-<date as YYYY-mm-DD>-RNv<version-of-react-native>`. For example: `hermesV1-2025-11-03-RNv0.83.0`. Do not add he prerelease tag
+
+### Step 4: Bump the Hermes version on the React Native release branch
+
+Using the newly generated Hermes tag run the following script on the React Native release branch:
+
+```bash
+# Replace <the_hermes_tag> with the tag that will look like 'hermes-2022-07-20-RNv0.70.0-bc97c5399e0789c0a323f8e1431986e207a9e8ba'
+./packages/react-native/scripts/hermes/bump-hermes-version.js -t <the_hermes_tag> -s <the_hermes_v1_tag> -h <the_hermes_version> -v <the_hermes_v1_version>
+```
+
+An example of the invocation is:
+```
+./packages/react-native/scripts/hermes/bump-hermes-version.js -t hermes-2025-11-03-RNv0.83.0 -s hermesV1-2025-11-03-RNv0.83.0 -h 250829098.0.2 -v 250829098.0.2
```

Comment:
The assumption here was that the versioning would be decoupled, so it would be something like

```suggestion
./packages/react-native/scripts/hermes/bump-hermes-version.js -t v0.14.0 -s v250829098.0.2 -h 0.14.0 -v 250829098.0.2
```

And as a sidenote - when `-h (legacy hermes version)` or `-v (hermes v1 version)` is skipped, the script will fetch the latest versions from NPM and prompt the caller whether they are correct.

If we assume that the tag is always in the format of `v$versionName`, then we could skip setting tags and infer them from passed versions (with an option to override if inferred wrongly).